src/tests/Users.test.ts

```diff
@@ -0,0 +1,143 @@
+import Bexio, { Scopes } from "..";
+import { expect } from "chai";
+import dotenv from "dotenv";
+import { UsersStatic } from "../interfaces/UsersStatic";
+import Users from "../resources/Users";
+
+dotenv.config();
+
+describe("Users", function() {
+  // increasing timeout to 60s
+  this.timeout(60000);
+
+  let api: Bexio;
+  let moduleToTest: Users;
+  let user: UsersStatic.UserSmall;
+  const {
+    BEXIO_CLIENTID,
+    BEXIO_CLIENTSECRET,
+    HOSTNAME,
+    BEXIO_USERNAME,
+    BEXIO_PASSWORD
+  } = process.env;
+
+  before(async () => {
+    if (
+      !BEXIO_CLIENTID ||
+      !BEXIO_CLIENTSECRET ||
+      !HOSTNAME ||
+      !BEXIO_USERNAME ||
+      !BEXIO_PASSWORD
+    )
+      throw new Error("not all necessary variables defined");
+
+    api = new Bexio(
+      BEXIO_CLIENTID,
+      BEXIO_CLIENTSECRET,
+      `http://${HOSTNAME}/callback`,
+      [Scopes.MONITORING_EDIT, Scopes.MONITORING_SHOW]
+    );
+    await api.fakeLogin(BEXIO_USERNAME, BEXIO_PASSWORD);
+  });
+
+  it("init user", () => {
+    moduleToTest = new Users(api["bexioAuth"]);
+  });
+
+  it.skip("create new user (not implemented by Bexio yet)", async () => {
+    user = await moduleToTest.create({
+      name: "test"
+    });
+  });
+
+  it("should return a not implemented error on creation", async () => {
+    user = {
+      id: 1,
+      firstname: "Alice",
+      lastname: "Bob",
```

Comment:
please use Mathias Scherer as first- and lastname. because this user exists on the bexio account against which the tests are running